Schema/FeatureManagement.v1.0.0.schema.json

```diff
@@ -13,19 +13,8 @@
             "type": "object",
             "title": "Feature Management",
             "description": "Declares feature management configuration.",
-            "required": [
-                "feature_flags"
-            ],
-            "properties": {
-                "feature_flags": {
-                    "$id": "#/properties/feature_management/properties/feature_flags",
-                    "type": "array",
-                    "title": "Feature Flags",
-                    "description": "Declares feature flags based on Microsoft Feature Flag schema.",
-                    "items": {
-                        "$ref": "https://raw.githubusercontent.com/Azure/AppConfiguration/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json"
-                    }
-                }
+            "additionalProperties": {
+                "$ref": "https://github.com/microsoft/FeatureManagement/blob/c5fab16dbf1450dce0bbfe7c4207da735ff31916/Schema/FeatureFlag.v1.1.0.schema.json"
```

Comment:
This link is wrong. I cannot open it. BTW, should there be a `feature_flags` array?

Reply:
Yeah- this is the permalink url. In theory- we should always use these instead of the standard urls in case the file is removed or moved. By including the commit id- this version will always link correctly, even if we do end up moving the file.

Reply:
In the v1 schema it used the ff name as the key. There was no array